### ISSUE TITLE
The method run_db_maintenance should be executed periodically to avoid big ncpa.db file

### ISSUE THREAD

**Issue by ccztux** [NagiosEnterprises/ncpa#865]
Actually the method `run_db_maintenance` is called only once at the start phase of the `ncpa_listener` and `ncpa_passive` services:

https://github.com/NagiosEnterprises/ncpa/blob/9e06ef7e0738eb7ac6f8b5a832e0a679ab3630d6/agent/listener/database.py#L47-L57

This behaviour can lead to big `ncpa.db` files. This could be avoided if this method would be exectuted at least once a day.

**Comment by jomann09** [comment]
I agree, I think this would be something we could bundle into NCPA 3 at some point since we will be running a sub process that runs all the time doing jobs and can do a scheduled job like this.
